evm/src/main/java/org/hyperledger/besu/evm/frame/MessageFrame.java

```diff
@@ -929,6 +929,17 @@ public boolean wasCreatedInTransaction(final Address address) {
     return txValues.creates().contains((address));
   }
 
+  /**
+   * Checks if there is an account for this address in the world state. This is different from
+   * `wasCreatedInTransaction` since an account could have been created in another transaction.
+   *
+   * @param address the address to check
+   * @return true if account already exists in world state
+   */
+  public boolean accountExists(final Address address) {
+    return worldUpdater.get(address) != null;
+  }
+
```

Comment:
Unused, I would remove

Reply:
not sure why it's here 🤔 I will remove it